doc/mainpage.dox.in

```diff
@@ -100,13 +86,14 @@ complete matrix stored as a distributed table.
         @defgroup grp_matrix Matrix Operations
 
         @defgroup grp_matrix_factorization Matrix Factorization
-        @brief Matrix Factorization methods including Singular Value Decomposition and Low-rank Matrix Factorization
+        Linear algebra methods that factorize a matrix
```

Comment:
Is it possible to keep two copies here, with one has @brief and the other one without brief so that they show up in both places in webpage?

Reply:
yes of course